File: javascript/natural.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
print("Enter a number upto which you would like to print natural number");
Count = readln();
Natural_numbers = "";
for (Counter = 1; Counter <= Count; Counter++)
{
  Natural_numbers += " " + Counter;
}
print("The first " + Count + " natural numbers are " + Natural_numbers + ".");

File: c/natural.c
#include <stdio.h>

int main(void) {
  int Count, Counter;
  printf("Enter the number upto which you want to print natural numbers : ");
  scanf("%d", &Count);
  printf("The natural numbers upto %d are", Count);
  for(Counter = 1; Counter <= Count; Counter++) {
    printf(" %d", Counter);
  }
  printf(".\n");
 
  return 0;
}